8/8.2/8.2/main.cpp

```diff
@@ -0,0 +1,53 @@
+#include <iostream>
+
+#include "string.hpp"
+using namespace std;
+
+int main()
```

Comment:
HEAP SUMMARY:
  in use at exit: 73,737 bytes in 5 blocks
total heap usage: 16 allocs, 11 frees, 84,013 bytes allocated